config/initializers/devise.rb

```diff
@@ -5,14 +5,6 @@
   # random tokens. Changing this key will render invalid all existing
   # confirmation, reset password and unlock tokens in the database.
   config.secret_key = ENV['DEVISE_SECRET_KEY']
-  # ==> Mailer Configuration
```

Comment:
When I went to edit this initializer file, I saw a **lot** of commented-out cruft configuration that doesn't apply to the Student Insights use case for Devise. 

I decided to remove a bunch of the commented-out material, since it's not relevant to our app and it makes it harder to discover the sections that are relevant.